README.md

```diff
@@ -11,17 +12,9 @@ recommendations to the platform. Its code can be found
 If you are interested in contributing, please checkout the information below
 and post an issue if you have any questions or would like to chat.
 
-## References
-
-- [Creating an issue related to deep](docs/create-issue.md)
-- [Contributing - Getting Started](docs/getting-started.md)
-- [Contributing - Frontend Details](docs/contributing_frontend.md)
-- [Contributing - Backend Details](docs/contributing_backend.md)
-- [DEEP Git For n00bz](docs/git.md)
-- [REST API](docs/api-rest.md)
-- [Websocket API](docs/api-websocket.md)
-- [Testing... Frontend](docs/testing_frontend.md)
-- [Testing... Backend](docs/testing_backend.md)
+## Deep Documentation
+
+- [Deep Documentation](https://docs.thedeep.io)
 
 ## Staging Servers
 
```

Comment:
We missed this. Let's update this to
```md
## Staging Servers

- [Webapp](https://staging.thedeep.io)
- [API](https://staging-api.thedeep.io)
```

Reply:
Let's remove the existing alpha instance from here as well. You can use the above as a whole replacement.